edwin/client/static/js/components/Timeline.js

```diff
@@ -13,6 +14,8 @@ import UserStore from '../stores/UserStore.js';
 import {BugStates} from '../constants/TimelineConstants';
 import TimelineActions from '../actions/TimelineActions.js';
 import TimelineConstants from '../constants/TimelineConstants.js';
+import IndicatorActions from '../actions/IndicatorActions.js';
+import {LoadingStates} from '../constants/IndicatorConstants';
```

Comment:
On a side note: These imports are getting out of control. This isn't your fault, and it doesn't need to change here, but I want to find a better way to deal with this. At least some standard for splitting these into groups and ordering them. Maybe some reduction of import lines some how. I'm not sure.